### DISCUSSION TITLE
How to set multi values Attribute to a new note from script

### DISCUSSION THREAD

**Discussion by PhilB-RLB** [TriliumNext/Trilium#2451] [Q&A] [answered]
Hello,

by using this following code :
for (label of Form_Note.getLabels("Artiste_Album")) {
   newNoteObject.setLabel("Artiste_Album(inheritable)", label.value);
}

If "Artiste_Album" contain the following values :
Artiste_1
Artiste_2
Artiste_3

only the last value (Artiste_3) of the multi values "Artiste_Album" is set to the new note (newNoteObject).

How to set ALL the 3 values (Artiste_1 to 3) to the new note ?

**Comment by zadam** [comment]
`.setLabel()` is used for single-valued attributes.

For multivalued attributes use `addLabel(name, value, isInheritable)`

Note that if you want to make it inheritable then you need to pass 3rd param as true and leave out that "(inheritable)" from name.

**Comment by PhilB-RLB** [comment, marked as answer]
Thank you !!!
